device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -333,10 +333,10 @@ public void close() throws TransportException
             }
             catch (InterruptedException e)
             {
-                throw new TransportException("Interrupted while closing proton reactor", e);
+                // If the SDK's disconnect message doesn't make it to the service, just clean up the local resources.
+                // The service will eventually figure out that the connection was lost since the keep-alive pings stop.
```

Comment:
I recently learned that http also supports keep-alive values and uses it for tls renegotiation. Do you know if we set http keep-alive in our sdk?

Reply:
I don't believe we do in the Java SDK, no